assets/css/style.css

```diff
@@ -8,6 +8,25 @@ body {
 	max-width: 1592px;
 	margin: 0 auto;
 }
+
+#loading{
```

Comment:
Don't use the loader till it finishes loading
Instead have it till the complete page loads
[Use this](https://smallenvelop.com/display-loading-icon-page-loads-completely/)

Reply:
okay